templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Do you want to delete {{ manufacturer.name }}?</h1>
+  
+  {% if manufacturer.cars.all %}
```

Comment:
The conditional check `if manufacturer.cars.all` should be `if manufacturer.cars.exists()`. The `exists()` method is more efficient for checking if there are any related cars.